server/package.json

```diff
@@ -4,15 +4,22 @@
   "description": "This respository consists all server side code",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon --exec babel-node src/index.js",
-    "test": "jest __tests__",
+    "start": "nodemon --exec babel-node src/app/index.js",
+    "test": "jest --forceExit --runInBand",
```

Comment:
why forceExit?

Reply:
Jest does not close the express server after every test case, so whenever the new request is sent to the express server, it shows the port is already in use, so I use --forceExit to close the server every time.

Reply:
@adityavyas611 I would like to add some points here. While testing using supertest you don't need to run the server. Try splitting the app and listening code separately. This worked for me.